src/test/java/ua/hillel/ContactUsTest.java

```diff
@@ -0,0 +1,45 @@
+package ua.hillel;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.Test;
+import ua.hillel.pages.ContactUsPage;
+import ua.hillel.pages.MainPage;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Properties;
+
+public class ContactUsTest {
+    @Test
+    public static void main(String[] args) throws InterruptedException {
```

Comment:
Try to name method in more "test" way, like:
```suggestion
    public void checkContactUsForm() throws InterruptedException {
```